pkg/syntheticmonitoring/synthetic-monitoring-handler.go

```diff
@@ -211,7 +211,7 @@ func (h *SyntheticMonitoringHandler) getRemoteCheck(uid string) (*grizzly.Resour
 			for _, probeID := range check.Probes {
 				probeNames = append(probeNames, probes.ByID[probeID].Name)
 			}
-			handler := SyntheticMonitoringHandler{}
+			handler := NewSyntheticMonitoringHandler(h.Provider)
```

Comment:
I feel like we should use the current struct (`h`) rather than create a new handler

Reply:
I see, made the change.